scot/eegtopo/geo_spherical.py

```diff
@@ -41,6 +41,11 @@ def vector(self):
         """position in 3d space"""
         return self._pos3d
 
+    @property
```

Comment:
I'm afraid the honest answer is because this is the first Python code I ever wrote. Thought it would be a good idea to make this a property.

Improving the legacy code is a task for a separate PR. Unfortunately a few unrelated commits in seaked in here.